browser/js/common/directives/navbar/navbar.html

```diff
@@ -106,9 +98,9 @@
 	<div ng-class='class'>
 		<div class='dropdown-lists'>
 
-			<div class='dropdown'>
-				<div class='dropdown-title' ng-click='goToSection(".overview", "home")'>What is Kinetic</div>
-				<div class='dropdown-content'>
+			<div class='dropdown' ng-init='whatIs = true'>
+				<div class='dropdown-title' ng-click="whatIs = !whatIs; getInvolved = true; resources = true; ourTeam = true">What is Kinetic</div>
+				<div ng-class="{hide: whatIs, show: !whatIs}">
```

Comment:
I'm not a huge fan of naming these variables solely after their contents. I'd make it something like `shouldShowWhatIs` (but even that I don't like all that much). See if you can come up with a more straightforward naming convention.